site/_data/cli/pulsar-admin.yaml

```diff
@@ -510,6 +510,164 @@ commands:
     argument: cluster-name
   - name: delete
     description: Delete namespace isolation policy of a cluster. This operation requires superuser privileges.
+- name: sink
+  description: An interface for managing Pulsar IO sinks (egree data from Pulsar)
+  subcommands:
+  - name: create
+    description: Create a Pulsar IO sink connector
+    options:
+    - flags: --className
+      description: The sink's Java class name
+    - flags: --cpu
+      description: The CPU (in cores) that needs to be allocated per sink instance (applicable only to the Docker runtime)
+    - flags: --customSerdeInputs
+      description: The map of input topics to SerDe class names (as a JSON string)
+    - flags: --disk
+      description: The disk (in bytes) that needs to be allocated per sink instance (applicable only to the Docker runtime)
+    - flags: --inputs
+      description: The sink's input topic(s) (multiple topics can be specified as a comma-separated list)
+    - flags: --jar
+      description: Path to the Java jar file for the sink
+    - flags: --name
+      description: The sink's name
+    - flags: --namespace
+      description: The sink's namespace
+    - flags: --parallelism
+      description: |
+        "The sink's parallelism factor (i.e. the number of sink instances to run)."
+    - flags: --processingGuarantees
```

Comment:
Thinking about it, it looks like processing guarantees make sense for functions, but for sources/sinks we might want to call it different?

Reply:
messagingGuarantees?

Reply:
if we decide to change it, lets to do that as part of a separate PR